### PR TITLE
Storage get count

### PR DESCRIPTION
**Pull request by lgonzalescc**
Hello Jairo this is the new pull request about README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,10 +153,12 @@ EOF  all  create  destroy  help  quit  show  update
 ## Bugs
 No known bugs at this time. 
 
-## Authors
+## Author
```
Comment (j4ir0st) [inline README.md]: Typo - fix Authors

**Review by j4ir0st — CHANGES_REQUESTED** [review]
Be carefull with the typos.

File: README.md (hunk 1 above)
Comment (lgonzalescc) [reply README.md]: Oh it's unfortunately I'm going to fix that

**Review by j4ir0st — APPROVED** [review]
Great job!

**Comment by j4ir0st** [comment]
Successfully fixed typos in README file

**Pull request merged**